deploy/kubernetes/releases/csi-digitalocean-dev/driver.yaml

```diff
@@ -150,12 +150,11 @@ spec:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: csi-resizer
-          image: registry.k8s.io/sig-storage/csi-resizer:v1.11.2
+          image: registry.k8s.io/sig-storage/csi-resizer:v1.12.0
```

Comment:
The latest version is actually 1.13.1, but in 1.13.1, `RecoverVolumeExpansionFailure` featuregate is enabled, and something about that feature isn't working with DO's csi. Investigating and fixing it will be a separate task.